packages/block-library/src/query/edit/inspector-controls/enhanced-pagination-control.js

```diff
@@ -25,19 +25,19 @@ export default function EnhancedPaginationControl( {
 		);
 	} else if ( enhancedPagination ) {
 		help = __(
-			"Browsing between pages won't require a full page reload, unless non-compatible blocks are detected."
+			'Reload the full page—instead of just the posts list—when visitors navigate between pages.'
 		);
 	} else if ( hasUnsupportedBlocks ) {
 		help = __(
-			"Force page reload can't be disabled because there are non-compatible blocks inside the Query block."
+			"Reload full page can't be disabled because there are non-compatible blocks inside the Query block."
```

Comment:
Do you think we need to repeat the name in this help text given the label is visible just above? It feels a bit duplicative to me.

Reply:
See also https://github.com/WordPress/gutenberg/pull/65081#discussion_r1745817225. I would be in favor of chaning it to 
```suggestion
			"Enhancement disabled because there are non-compatible blocks inside the Query block."
```